src/app/component/comments/services/comments.service.ts

```diff
@@ -29,6 +29,9 @@ export class CommentsService {
     return this.http.get(`${this.backEnd}econews/comments?ecoNewsId=${this.ecoNewsId}&page=0&size=12`);
   }
 
+  public getAllReplies(id: number): Observable<object> {
+    return this.http.get(`https://greencity.azurewebsites.net/econews/comments/replies/${id}`);
```

Comment:
please add here type and use alias for link

Reply:
> please add here type and use alias for link

Fixed